pkg/intermediate/aggregate.go

```diff
@@ -215,19 +215,19 @@ func (a *AggregationProcess) GetExpiryFromExpirePriorityQueue() time.Duration {
 	return a.inactiveExpiryTimeout
 }
 
-// GetRecords returns the string format flow records by FlowKey from flowKeyRecordMap.
-// Returns partial match if FlowKey is not complete.
-// Returns all the flow records if FlowKey is not provided.
-func (a *AggregationProcess) GetRecords(flowKey *FlowKey) []string {
+// GetRecords returns map format flow records given a flow key.
```

Comment:
Let's explain the map format clearly, where key is element name and value is the IE object.

Reply:
Done